### PR TITLE
Use _.create in extend

### PR DESCRIPTION
**Pull request by megawac**
As mentioned here https://github.com/jashkenas/underscore/pull/2074

### CODE REVIEW

**Comment by jashkenas** [comment]
Is it worth breaking backwards compatibility so soon for this? Not worth waiting until `_.create` has a chance to penetrate for a while?

**Comment by megawac** [comment]
Just raising an issue, can sit on it. It shouldn't break backwards compat unless people are relying on old versions of underscore

**Comment by jridgewell** [comment]
The extend vs assign (in create) is a midly breaking change. I think the common case, though, is simple objects.

File: backbone.js [hunk 1]
```diff
@@ -1793,14 +1793,8 @@
...
-    // `parent`'s constructor function.
-    var Surrogate = function(){ this.constructor = child; };
-    Surrogate.prototype = parent.prototype;
-    child.prototype = new Surrogate;
-
-    // Add prototype properties (instance properties) to the subclass,
-    // if supplied.
-    if (protoProps) _.extend(child.prototype, protoProps);
+    // `parent`'s constructor function and add the prototype properties.
+    child.prototype = _.create(parent.prototype, protoProps);
```
Comment (jridgewell) [inline backbone.js]: You'll also need to set `child.prototype.constructor = child`.

**Comment by jridgewell** [comment]
Ping @megawac, want to rebase and add in the `child.prototype.constructor = child` change?

**Comment by megawac** [comment]
Right, done

**Comment by jridgewell** [comment]
LGTM.

**Comment by akre54** [comment]
April's long enough. Let's do this.

**Pull request merged**